Description: better comment
 flush2.test fails on 6C12T Ryzen probably because of a buggy Perl test
 framework. This failure has nothing to do with the tested behaviour it
 seems. This is a start of investigation.
 .
 mysql-5.7 (5.7.29-u20.04.0) focal; urgency=medium
 .
   * Non-maintainer upload.
   * rebuild for focal
Author: Mark Zhitomirski <mz@exactpro.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Forwarded: not-needed
Last-Update: 2020-04-19

--- mysql-5.7-5.7.29.orig/mysql-test/t/flush2.test
+++ mysql-5.7-5.7.29/mysql-test/t/flush2.test
@@ -1,6 +1,8 @@
 
 #
-# Bug#17733 Flushing logs causes daily server crash
+# Bug#17733 Flushing logs causes server crash
+# https://bugs.mysql.com/bug.php?id=17733
+# fixed in 5.0.33 2007-01-09
 #
 
 --source include/not_embedded.inc
